src/bower_components/emby-webcomponents/nowplayingbar/nowplayingbar.css

```diff
@@ -119,38 +119,75 @@
     height: 1.2em !important;
 }
 
-@media all and (max-width: 87.5em) {
+@media all and (max-width: 73em) {
 
-    .nowPlayingBarUserDataButtons {
+    .nowPlayingBarRight .nowPlayingBarUserDataButtons {
         display: none;
     }
 }
 
-@media all and (max-width: 68.75em) {
+@media all and (max-width: 70em) {
 
-    .nowPlayingBarVolumeSliderContainer, .nowPlayingBar .muteButton, .nowPlayingBar .unmuteButton {
+    .nowPlayingBarRight .toggleRepeatButton {
+        display: none;
+    }
+}
+
+@media all and (max-width: 67em) {
+
+    .nowPlayingBarCenter .nowPlayingBarCurrentTime {
         display: none !important;
     }
+
 }
 
-@media all and (max-width: 50em) {
+@media all and (max-width: 61em) {
 
-    .nowPlayingBarCenter {
+    .nowPlayingBarCenter .stopButton {
         display: none !important;
     }
 
-    .toggleRepeatButton {
-        display: none;
+}
+
+
+@media all and (max-width: 55em) {
+
+    .nowPlayingBarCenter .nextTrackButton, .nowPlayingBarCenter .previousTrackButton, .nowPlayingBarCenter .playPauseButton {
+        display: none !important;
```

Comment:
did you try without the !important? can't see to find more classes that use this sub classes with a display none important.

Also, nowPlayingBarCenter is parent of some of those classes. Try removing all sub classes of this one.

Reply:
forget that last part, you are applying the rules to the sub classes, not the parent one.

Reply:
I never myself use !important. That is a residue of the old code. I'll try without, it is a good idea.